### DISCUSSION TITLE
Parallel Plotting

### DISCUSSION THREAD

**Discussion by cexmad** [Chia-Network/chia-blockchain#5161] [Support]
Hello,

I started parallel plotting yesterday. There were 6x process started and after unexpected restart i can not see any plotting process. Well they were at %30 and i got 1.1 TB plot temp files. 
I know i can not continue to those plotting process, i need to know that what should i do with these files? 
Should i move these files from SSD to SATA or is there any way to verify these files?
Thanks in advanced for your helps.

**Comment by keeperlink** [comment]
Delete all temp files and start over. There no resume where it stopped option

**Reply by cexmad** [reply]
Do you have official resource for this? Because deleting the 1.1 TB file created after 24 hours of labor is kind an absurd.

**Reply by keeperlink** [reply]
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#plotting :

My computer/drive went into power save or rebooted while plotting?
Unfortunately, resuming a plot is not yet supported but likely will be later in 2021. We suggest that you disable power saving mode - especially for external drives - and try to limit other possible causes of interruptions. Plotting a k=32 is going to take between 6 and 20 hours, depending on your hardware, so these interruptions can be painful. They are also a part of why we don't recommend plotting plots larger than k=32 as each increment in k generally doubles the time to complete a single plot.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
